interop/package.json

```diff
@@ -47,7 +47,8 @@
     "start": "node index.js",
     "build": "aegir build",
     "lint": "aegir lint",
-    "test:interop:multidim": "aegir test"
+    "test:interop:multidim": "aegir test",
```

Comment:
Do we still need this line?

Reply:
yes we do, in order to be backwards compatible until https://github.com/libp2p/test-plans/pull/312 is merged